app/poros/party_creator.rb

```diff
@@ -0,0 +1,17 @@
+class PartyCreator
+  def initialize(data, user, movie)
+    @party = data[:party]
+    @user = user
+    @movie = movie
+  end
+
+  def make_the_party
+    party = Party.create(time: @party[:time], duration: @party[:duration], user: @user, movie: @movie)
+    @party[:guests].each do |guest|
```

Comment:
Does this work because @party has_many guests? or is something else happening here.

Reply:
For whatever reason, the :guests params always passes 1 [""], and then any friends you check are added to that array.  I had to call it :guests vs invites in the view because a form_with on a model will error on unknown attribute.